python/cudf/cudf/utils/metadata/orc_column_statistics.proto

```diff
@@ -0,0 +1,61 @@
+message IntegerStatistics  {
```

Comment:
Add NVIDIA copyright to top of this file and additionally add a comment explaining how to use `protoc` to build this into the Python module

Reply:
Done